luskan/src/main/java/org/opencadc/luskan/LuskanConfig.java

```diff
@@ -169,6 +172,21 @@ public static MultiValuedProperties getConfig() {
         }
         sb.append("\n");
 
+        List<String> allowedGroups = props.getProperty(ALLOWED_GROUP);
+        if (allowedGroups.isEmpty()) {
+            sb.append("MISSING");
```

Comment:
Since this is effectively required, `ok = false`??

Reply:
Yes `ok=false` is missing in a couple of places.